crates/symbolicator/src/services/download/s3.rs

```diff
@@ -116,7 +116,11 @@ impl S3Downloader {
             );
             let s3 = Arc::new(match key.aws_credentials_provider {
                 AwsCredentialsProvider::Container => {
-                    let provider = rusoto_credential::ContainerProvider::new();
+                    let container_provider = rusoto_credential::ContainerProvider::new();
+                    let provider = match rusoto_credential::AutoRefreshingProvider::new(container_provider) {
+                        Ok(provider) => provider,
+                        Err(err) => panic!("Unable to instantiate rusoto_credential::AutoRefreshingProvider: {:?}", err),
```

Comment:
As source configs are being fed into symbolicator with each request, we should avoid panic-ing as much as possible. I haven’t looked at the docs to figure out the exact failure conditions for this `Result`, it might as well be an extreme edge-case condition in which case it might as well be okay to `unwrap`.

Reply:
Looking at the [source](https://rusoto.github.io/rusoto/src/rusoto_credential/lib.rs.html#243-251) of `rusoto_credential::AutoRefreshingProvider::new`, I don't even understand how it can return `CredentialsError`?